src/utils/TriangleUtils.ts

```diff
@@ -286,36 +498,16 @@ TriangleUtils.getParametrisedVertexAttr = function (triangle, u, v) {
     pos.addVectors(pos, vAdd);
 
     return { "pos": pos, "thick": meanThick };
-};
+},
 
 /**
- *  @param {!Object} triangle The concerned triangle
- *  @param {number} u u coordinate
- *  @param {number} v v coordinate
- *  @return {number}
+ *  @param triangle The concerned triangle
+ *  @param u u coordinate
+ *  @param v v coordinate
  */
-TriangleUtils.getMeanThick = function (triangle, u, v) {
+getMeanThick (triangle: TriangleLike, u: number, v: number): number {
     return triangle.v[0].getThickness() * (1 - u - v) + triangle.v[1].getThickness() * u + triangle.v[2].getThickness() * v;
-};
-
-/**
- *  @param {!Object} triangle The concerned triangle
- *  @param {number} u u coordinate
- *  @param {number} v v coordinate
- *  @return {!Material} Interpolated material
- */
-TriangleUtils.getMeanMat = function (triangle, u, v) {
```

Comment:
where did that go ?

Reply:
This function is misplaced and not used in this repository. Maxime told me to remove it